_data/domains.yml

```diff
@@ -111,6 +111,14 @@ websites:
       tfa: No
       status: https://twitter.com/enom/status/447074738033008640
 
+    - name: EuroDNS
+      url: https://www.eurodns.com
+      twitter: EuroDNS
```

Comment:
The twitter handle isn't needed when the site already supports TFA. Please remove it